src/Files.App/Services/PreviewPopupProviders/QuickLookProvider.cs

```diff
@@ -45,6 +45,10 @@ private async Task DoPreviewAsync(string path, string message)
 			{
 				client.Close();
 			}
+			catch (IOException)
```

Comment:
The following would allow for a single `catch` clause.
```csharp
catch (Exception ex) when (ex is TimeoutException or IOException)
```

Reply:
Thanks!